wemake_python_styleguide/visitors/tokenize/primitives.py

```diff
@@ -221,6 +224,15 @@ def _check_wrong_unicode_escape(self, token: tokenize.TokenInfo) -> None:
             # character can never be the start of a new backslash escape.
             index += 2
 
+    def _check_unnecessary_raw_string(self, token: tokenize.TokenInfo) -> None:
+        modifiers, string_def = split_prefixes(token)
+
+        if 'r' in modifiers.lower():
+            if '\\' not in string_def.encode('unicode-escape').decode():
```

Comment:
`r'\'` is better here.

Reply:
Can you please also explain `encode('unicode-escape').decode()` part? Why is it required?

Reply:
No, `r'\'` is not correct in Python I think.
```console
Python 3.6.9
>>> a=r'\'
  File "<stdin>", line 1
    a=r'\'
         ^
SyntaxError: EOL while scanning string literal
>>> 
```

The `encode('unicode-escape').decode()` is a workaround to convert into a raw string. ([see here](https://stackoverflow.com/a/58309907)). If you know something more beautiful let me know!

Reply:
Ok! 👍

Reply:
@Mema5 have you tried running tests without `encode('unicode-escape').decode()`? Did it work?